app/services/transaction.service.js

```diff
@@ -22,10 +23,9 @@ const connectorRefundFailureReasons = {
 
 const searchLedger = async function searchLedger (gatewayAccountIds = [], filters) {
   try {
-    const transactions = await Ledger.transactions(gatewayAccountIds, filters)
-    return transactions
+    return await Ledger.transactions(gatewayAccountIds, filters)
   } catch (error) {
-    throw new Error('GET_FAILED')
+    throw handleErrorForFailedSearch(error)
```

Comment:
It seems that this method can take a `response` parameter but we’re not passing one.